c/csv/writer.cc

```diff
@@ -464,6 +464,7 @@ size_t CsvWriter::estimate_output_size()
   size_t ncols = static_cast<size_t>(dt->ncols);
   size_t total_string_size = 0;
   size_t total_columns_size = 0;
+  fixed_size_per_row = ncols;  // 1 byte per separator
```

Comment:
Do we also need to add 1 byte for `\n`?

Reply:
There are `ncols-1` separators + 1 newline, `ncols` bytes in total.